pkg/config/schema/metadata.yaml

```diff
@@ -121,6 +121,25 @@ resources:
     specless: true
     proto: "k8s.io.api.core.v1.ServiceAccount"
     protoPackage: "k8s.io/api/core/v1"
+  
+  - kind: "ServiceImport"
+    plural: "serviceimports"
+    group: "multicluster.x-k8s.io"
+    version: "v1alpha1"
+    protoPackage: "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
+    proto: "k8s.io.mcs_api.pkg.apis.v1alpha1.ServiceImportSpec"
+    statusProto: "k8s.io.mcs_api.pkg.apis.v1alpha1.ServiceImportStatus"
+    statusProtoPackage: "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
+
+  - kind: "ServiceExport"
```

Comment:
MCS was intentionally not using this and using MCS_API_GROUP instead, as there was (at least at the time), a bunch of vendor specific MCS type mirrors since MCS was alpha.